pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -578,6 +600,16 @@ func (t *Task) areStunnelClientPodsRunning() (bool, error) {
 		}
 		for _, pod := range pods.Items {
 			if pod.Status.Phase != corev1.PodRunning {
+				for _, podCond := range pod.Status.Conditions {
+					if podCond.Reason == corev1.PodReasonUnschedulable {
+						t.Log.Info(fmt.Sprintf("Found UNSCHEDULABLE Stunnel Client Pod [%v/%v] "+
```

Comment:
Added this check that logs reason if pods are unschedulable. I ran into this a few times while testing out new logging.